project.json

```diff
@@ -84,6 +84,13 @@
         ],
         "scripts": []
       }
+    },
+    "write-json": {
+      "executor": "nx:run-commands",
+      "options": {
+        "command": "ts-node tools/src/write-json.ts dist/ngx-libs/assets/libs.json"
+      },
+      "dependsOn": ["build"]
```

Comment:
I wanted to do it the opposite way (build depending on write-json) but I need the generated dist folder to write the json in.